keys/keys.go

```diff
@@ -53,6 +58,13 @@ func (Storable) TableName() string {
 	return "storable_keys"
 }
 
+type Proposer struct {
+	ID        int `json:"-" gorm:"primaryKey"`
+	KeyIndex  int `gorm:"unique"`
+	CreatedAt time.Time
+	UpdatedAt time.Time
+}
```

Comment:
Maybe add a `TableName()` method to keep the SQL clean:

```
func (ProposalKey) TableName() string {
	return "proposal_keys"
}
```

Reply:
Good call, added https://github.com/onflow/wallet-api/pull/13/commits/bb9c04463bda82301e82f56be469352438aa3811